Update Service Insight docs (#3684)
nginxinc/kubernetes-ingress: docs/content/logging-and-monitoring/service-insight.md

---

> The Service Insight is available only for NGINX Plus. The Ingress Controller exposes an endpoint and provides host statistics for services exposed using the  VirtualServer (VS) and TransportServer (TS) resource objects.

Reviewer: ```suggestion
Service Insights are available only for NGINX Plus. The NGINX Ingress Controller exposes an endpoint which provides host statistics for services exposed using the VirtualServer (VS) and TransportServer (TS) resources
```

> The Service Insight feature is available only for NGINX Plus. The NGINX Ingress Controller exposes an endpoint which provides host statistics for services exposed using the VirtualServer (VS) and TransportServer (TS) resources.

Reviewer 2: ```suggestion
The Service Insight feature is available only for F5 NGINX Plus. The F5 NGINX Ingress Controller exposes an endpoint which provides host statistics for services exposed using the VirtualServer (VS) and TransportServer (TS) resources.
```
